ARC specification.md

```diff
@@ -369,64 +369,50 @@ This process is further referred to as _Continuous Quality Control_ (CQC) of the
+
 - MUST be named `cqc`
 - MUST be an [orphan branch](https://git-scm.com/docs/git-checkout#Documentation/git-checkout.txt---orphanltnew-branchgt)
 - MUST NOT be merged into any other branch. 
 - MUST contain the following folder structure:
 
-  `{$branch}/{$commithash}/{$package}`:
+  `{$branch}/{$package}`:
 
   ```
   cqc branch root
   └── {$branch}
-      └── {$commithash}
-          └── {$package}
+      └── {$package}
   ```
   
   where:
   - `{$branch}` is the name of the branch the validation was run on
-  - `{$commithash}` is the full hash of the commit the validation was run on. 
   - `{$package}` is the name of the validation package the validation was run against. 
     this folder then MUST contain the files `validation_report.*` and `badge.svg` as described in the [validation package specification](#validation-packages).
+    This folder MAY also be suffixed by the version of the validation package via a `@` character followed by the version number of the validation package: `{$package}@{$version}`, e.g. `package1@1.0.0`.
 
   example:
 
-  > This example shows the validation results of the `main` and `branch-1` branches of the ARC repository against the `package1` and `package2` validation packages for two commits per branch:
+  > This example shows the validation results of the `main` and `branch-1` branches of the ARC repository against the `package1` and `package2` validation packages. for `package2`, an optional version hint of the package is included in the folder name:
 
   ```
   cqc-branch-root
   ├── branch-1
-  │   ├── ca82a6dff817ec66f44342007202690a93763949
-  │   │   ├── package1
-  │   │   │   ├── badge.svg
-  │   │   │   └── validation_report.xml
-  │   │   └── package2
-  │   │       ├── badge.svg
-  │   │       └── validation_report.xml
-  │   └── 085bb3bcb608e1e8451d4b2432f8ecbe6306e7e7
-  │       ├── package1
-  │       │   ├── badge.svg
-  │       │   └── validation_report.xml
-  │       └── package2
-  │           ├── badge.svg
-  │           └── validation_report.xml
+  │   ├── package1
+  │   │   ├── badge.svg
+  │   │   └── validation_report.xml
+  │   └── package2@2.0.0
+  │       ├── badge.svg
+  │       └── validation_report.xml
   └── main
-      ├── 1234567890abcdef1234567890abcdef12345678
-      │   ├── package1
-      │   │   ├── badge.svg
-      │   │   └── validation_report.xml
-      │   └── package2
-      │       ├── badge.svg
-      │       └── validation_report.xml
-      └── a11bef06a3f659402fe7563abf99ad00de2209e6
-          ├── package1
-          │   ├── badge.svg
-          │   └── validation_report.xml
-          └── package2
-              ├── badge.svg
-              └── validation_report.xml
+      ├── package1
+      │   ├── badge.svg
+      │   └── validation_report.xml
+      └── package2@2.0.0
+          ├── badge.svg
+          └── validation_report.xml
   ```
 
+Commits to the `cqc` branch MUST contain the commit hash of the commit that was validated in the commit message.
```

Comment:
Can you elaborate on this?

Reply:
You mean here or in the document? 😆 

What i mean with this is that the system implementing the commits to the cqc branch must include the hash of the commit that was validated. not sure how to make that clearer.

Reply:
>What i mean with this is that the system implementing the commits to the cqc branch must include the hash of the commit that was validated in the commit message that adds the validation results. not sure how to make that clearer.

Ahh you mean the commit hash of the validated commit must be included in the commit `message` in the cqc branch?

If yes, maybe include the word `message` in the specs.

Reply:
> OTOH this might lead to huge commit messages that contain the commit hashes when many branches are validated, maybe leave this out or change to SHOULD?

Is it possible that multiple branches get validated in parallel?

Reply:
>Ahh you mean the commit hash of the validated commit must be included in the commit message in the cqc branch?
>
> If yes, maybe include the word message in the specs.

i mean it is right there:

> Commits to the `cqc` branch MUST contain the commit hash of the commit that was validated in the **commit message.**

> Is it possible that multiple branches get validated in parallel?

You mean from a technical standpoint or according to spec? It is possible, hence the `$branch` in the folder path `{$branch}/{$package}` in the cqc branch specs. However, i guess from a technical standpoint, validating each branch and then committing the results per branch to cqc is easier. This would mean that commit messages to `cqc` would have the format `{$branch}/{$commithash}`, where `$branch` is the validated branch and `$commithash` is the hash of HEAD commit on `$branch`. would that be clearer?

Reply:
Whoops, somehow didn't parse the `message` at the end. But I really struggled to understand that sentence without the extra discussion here. So maybe would still be beneficial to split that sentence into two, decreasing information density in exchange for clarity.

Reply:
> You mean from a technical standpoint or according to spec? It is possible, hence the $branch in the folder path {$branch}/{$package} in the cqc branch specs. However, i guess from a technical standpoint, validating each branch and then committing the results per branch to cqc is easier. This would mean that commit messages to cqc would have the format {$branch}/{$commithash}, where $branch is the validated branch and $commithash is the hash of HEAD commit on $branch. would that be clearer?

Not sure what would be the best solution. I'm just wondering whether there is any benefit of having cqc on different branches on the same commit hash, as these branches are structurally equal. In theory the branch doesn't really matter at all besides human readability. The only important bit is the hash.